src/rer/customersatisfaction/browser/configure.zcml

```diff
@@ -19,6 +19,12 @@
       directory="static"
       />
 
+  <plone:static
+      directory="idler_apps/dist"
```

Comment:
userei la cartella static che già era stata definita.
Anche perché l'url con ++plone++idler_apps non mi sta a significare nulla, mentre ++plone++rer.customersatisfaction ha senso e fa capire a cosa serve quel js che si sta scaricando.

In generale sposterei quello che hai fatto dentro idler_apps in static

Reply:
done